### ISSUE TITLE
Cannot read property 'call' of undefined

### ISSUE THREAD

**Issue by GeekBroadway** [DoctorMcKay/node-steam-user#136]
Hello, I recently started getting this with code unchanged. 

Node Version: 6.11.3
```javascript
/home/vagrant/steamitembot/node_modules/steam-user/components/messages.js:231
        this._handlers[handlerName].call(this, body, cb);
                                   ^

TypeError: Cannot read property 'call' of undefined
    at SteamUser._handleMessage (/home/vagrant/steamitembot/node_modules/steam-user/components/messages.js:231:29)
    at emitThree (events.js:116:13)
    at CMClient.emit (events.js:194:7)
    at CMClient._netMsgReceived (/home/vagrant/steamitembot/node_modules/steam-client/lib/cm_client.js:323:8)
    at CMClient.handlers.(anonymous function) (/home/vagrant/steamitembot/node_modules/steam-client/lib/cm_client.js:603:8)
    at CMClient._netMsgReceived (/home/vagrant/steamitembot/node_modules/steam-client/lib/cm_client.js:305:24)
    at emitOne (events.js:96:13)
    at TCPConnection.emit (events.js:188:7)
    at TCPConnection._readPacket (/home/vagrant/steamitembot/node_modules/steam-client/lib/tcp_connection.js:179:7)
    at emitNone (events.js:86:13)
```
To further test, I downloaded the example basicbot.js and got the same error when running it.

**Comment by DoctorMcKay** [comment]
Fixed in v3.23.1 which I've published to npm just now.

**Comment by GeekBroadway** [comment]
You are a legend
